root/src/emails/send_membership_email/send_membership_email.txt

```diff
@@ -18,7 +18,7 @@ Please pop along to the next Wednesday open evening or message us on telegram to
 Please do make sure you have read the Member's Guide (which you just
 agreed to!) as this details how the space works - if you missed it,
 here is the link again:
-https://docs.google.com/document/d/1ruqYeKe7kMMnNzKh_LLo2aeoFufMfQsdX987iU6zgCI/edit#heading=h.a7vgchnwk02g
+https://docs.google.com/document/d/1aN5Av5Qw--z31qGUhOl-NAiisJi5eZLWK7v47Li5uFU/edit?usp=sharing
```

Comment:
This link, and all similar ones, end in /edit, which suggests that the link is to edit the document, which isn't what we want.  Change to "view" instead.  (However, google ignores this anyway, so it's a minor point.